dig_test.go

```diff
@@ -1009,6 +1009,7 @@ func TestProvideCycleFails(t *testing.T) {
 			In
 
 			B B
+			W io.Writer
```

Comment:
Accident or purposeful?

Reply:
Purposeful. Covers a path not previously covered.